### PR TITLE
Gold on the map (merge me?)

### PR DESCRIPTION
**Pull request by Haganeko**
Merge me?

### CODE REVIEW

**Comment by chrisinajar** [comment]
Wow! Talk about a nerf to supports! Lets see how it goes....

**Comment by Haganeko** [comment]
Also why is it failing D=

File: game/scripts/npc/items/farming/item_greater_guardian_greaves_4.txt [hunk 1]
```diff
@@ -126,7 +126,6 @@
       {
         "var_type"                                        "FIELD_INTEGER"
         "replenish_mana"                                  "200 300 475 700"
-      }
```
Comment (chrisinajar) [inline game/scripts/npc/items/farming/item_greater_guardian_greaves_4.txt]: this is why

**Comment by chrisinajar** [comment]
is this designed for the new map or the old map?

**Comment by Haganeko** [comment]
This is for the new

**Comment by Haganeko** [comment]
Even rebased it atop warp's PR

**Comment by chrisinajar** [comment]
please never do that.

**Comment by chrisinajar** [comment]
fortunately you didn't actually rebase it on warp's PR :P

**Comment by warpdragon** [comment]
image

File: game/scripts/npc/items/farming/item_greater_guardian_greaves_4.txt (hunk 1 above)
Comment (imaGecko) [inline game/scripts/npc/items/farming/item_greater_guardian_greaves_4.txt]: ples fix

**Comment by Haganeko** [comment]
I actually did. This is from the same branch I PR'd to warp's repo. The reason those are not showing is this is a PR to test branch, which has warp's PR in it IIRC

**Comment by Haganeko** [comment]
Or maybe I tried and failed?

**Comment by chrisinajar** [comment]
Ah, you're right, next time don't do that. That's a super duper mega nono. This branch is unmergable with master now and no more mergable with test-client than before.

**Comment by chrisinajar** [comment]
this is a duplicate of another PR but with a messier history, closing in favor of the clean one.

**Pull request closed without merging**